src/Components/NavBar/Menus/MobileNavMenu.tsx

```diff
@@ -14,7 +14,7 @@ import {
 
 import { SystemContext } from "Artsy"
 import { useTracking } from "Artsy/Analytics/useTracking"
-import * as auth from "Components/NavBar/Utils/auth"
+import * as authentication from "Components/NavBar/Utils/authentication"
```

Comment:
Updated per feedback, but note that the language in Force is `auth`, which this was designed to match. ("Auth" as a language choice is also very common and unambiguous.)

Reply:
It’s true that it is common usage, but not unambiguous. Depending on the reader it will mean either authentication or authorization, or both. As many people don’t realize the important differences between authentication and authorization, removing ambiguity at any opportunity is imo a minimal step towards education of these concepts.

A random example from the many search results for this topic https://stackoverflow.com/questions/6556522/authentication-versus-authorization

Reply:
That makes sense, I didn't think about 'authorization' 👍